app/models/user.rb

```diff
@@ -31,8 +32,4 @@ def full_name
   def admin?
     false
   end
-
-#  def to_param
-#    slug
-#  end
```

Comment:
I left this in so we could have a conversation about it.
*I'm afraid it may be a little to complicated for my typing skills, or I'd throw down here.*